pkgs/development/compilers/ghc/8.2.1.nix

```diff
@@ -1,29 +1,58 @@
-{ stdenv, lib, fetchurl, bootPkgs, perl, ncurses, libiconv, binutils, coreutils
+{ stdenv, fetchurl, bootPkgs, perl, binutils, coreutils
 , autoconf, automake, happy, alex, python3, sphinx, hscolour
-, buildPlatform, targetPlatform , selfPkgs, cross ? null
-
-  # If enabled GHC will be build with the GPL-free but slower integer-simple
+, ncurses, gmp, libffi
+, __targetPackages
```

Comment:
This is technically something bound in allpackages, rather than a ghc-specific "configuration argument", but sure. Do note that in https://github.com/NixOS/nixpkgs/pull/26805 I add `__targetInputs` and `__propagatedTargetInputs` so there is no need for this.